### PR TITLE
[[ Scriptify revLibUrl ]] Ensure custom properties are initialised when ...

### PR DESCRIPTION
**Pull request by livecodeali**
...libUrl is loaded

### CODE REVIEW

File: ide-support/revliburl.livecodescript [hunk 1]
```diff
@@ -58,8 +58,25 @@ local laUrlByFile ##used by libUrlUploadFile to track which files go to which ur
...
 ------------------------------------------------
+on initialiseCustomProps
+   # initialise ftp codes prop set
+   local tFtpCodes
+   put "ABOR:225,226|Connect:220|CWD:250|DELE:250|LIST:125,150|MKD:257|MODE:200|NLST:125,150|PASS:230,202|PASV:227|PORT:200|PWD:257|QUIT:221|RETR:125,150|RMD:250|SIZE:213|STOR:125,150|transferComplete:226|TYPE:200|USER:230,331" into tFtpCodes
+   split tFtpCodes by "|" and ":"
+   set the customProperties["cFtpGoodCodes"] of me to tFtpGoodCodes
+   
+   # initialise default header
+   set the cDefaultHeader of me to "METHOD --- HTTP/1.1" & return & "Host:" & return & "User-Agent: "
```
Comment (livecodestephen) [inline ide-support/revliburl.livecodescript:74]: Should this be `crlf` instead of `return` as the HTTP standard uses crlf for line breaks in the header.

**Comment by runrevmark** [comment]
I'm pretty sure libURL does the conversion elsewhere... crlf and return are normalized to ascii 10 IIRC.

**Comment by livecodeali** [comment]
It must be done elsewhere as the value in the current custom property of the revLibUrl button also contains CRs rather than CRLFs.

**Comment by livecodeali** [comment]
This branch name was not really appropriate so this pull request is now here:
https://github.com/runrev/livecode/pull/1749

**Pull request closed without merging**